requests.go

```diff
@@ -92,8 +94,8 @@ type ReqPutAccountData struct {
 	Data map[string]interface{}
 }
 
-// ReqEmailRequestToken is the JSON request for
-// 		https://matrix.org/docs/spec/client_server/r0.6.1#post-matrix-client-r0-register-email-requesttoken
+//	 ReqEmailRequestToken is the JSON request for
```

Comment:
Was it formatted automatically?

Reply:
https://tip.golang.org/doc/go1.19#go-doc and might help

Reply:
If I just save `gofmt` changes the comments to this:
![image](https://user-images.githubusercontent.com/74783603/191192974-1bfb3063-b4bf-486a-be16-68f03dcc3980.png)
But if I add a single space then it turns to this here 😓 
Whatever you prefer 😆

Reply:
This one looks better to me :)

Reply:
@PiotrKozimor Done :)